.evergreen/config.in.yml

```diff
@@ -810,19 +810,25 @@ functions:
       params:
```

Comment:
since you didn't touch these lines it's non-blocking but should we update this function to use `subprocess.exec` instead of `shell.exec`?

Reply:
I may try to do that in the next PR since since this is working now don't want to tinker with it. :)